src/components/IfAllowed.js

```diff
@@ -0,0 +1,65 @@
+// flow
+
+import React from 'react';
+import type { Children } from 'react';
+import PropTypes from 'prop-types';
+
+import Configurable from './Configurable';
+
+type IfAllowedProps = {
+  /**
+   * The role representing a top-level admin, allowed to view any available widget.
+   * Optional, defaulting to "AIE_Administrator"
+   */
+  adminRole: string;
```

Comment:
I think we can just hard-code this.

Reply:
Okay,
I'll make it a static field in AuthUtils